tests/BaGetter.Core.Tests/Services/PackageIndexingServiceInMemoryTests.cs

```diff
@@ -117,9 +125,6 @@ public async Task IndexIMAsync_WhenPackageAlreadyExists_AndOverwriteAllowed_Inde
         var stream = new MemoryStream();
         builder.Save(stream);
 
-        _storage.Setup(s => s.DeleteAsync(builder.Id, builder.Version, default)).Returns(Task.CompletedTask);
-        _storage.Setup(s => s.SavePackageContentAsync(It.Is<Package>(p => p.Id == builder.Id && p.Version.ToString() == builder.Version.ToString()), stream, It.IsAny<FileStream>(), default, default, default)).Returns(Task.CompletedTask);
```

Comment:
Why did you remove the tests for Delete and Save?

Reply:
same as above